ui/values/dimensions.go

```diff
@@ -92,6 +92,7 @@ var (
 	MarginPadding500      = unit.Dp(500)
 	MarginPadding550      = unit.Dp(550)
 	MarginPadding600      = unit.Dp(600)
+	MarginPadding700      = unit.Dp(700)
```

Comment:
Consider using `DP700`, it's clearer and fits any use case.

Reply:
He should stay in line with the margin padding naming convention, it we want to change that, then we can refactor later

Reply:
ok.